src/components/Calendar/Calendar.cy.tsx

```diff
@@ -34,10 +35,16 @@ describe("<Calendar />", () => {
             end: testDate,
             allDay: false,
             description: "a piece of description text",
+            location: "Brixton Hill - Methodist Church",
         },
     ];
     it("calendar renders", () => {
-        cy.mount(<StyledCalendar initialEvents={[]} />);
+        cy.mount(
+            <StyledCalendar
+                initialEvents={[]}
+                initialLocations={["Brixton Hill - Methodist Church"]}
```

Comment:
can we move this into `StyledCalendar` (so we don't have to do this every time?)